__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed.js");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data");
+const db = require("../db/connection.js");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe.only("GET /api/topics", () => {
+  it("200: responds with an array of topic objects, each containing a slug and a description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toBeInstanceOf(Array);
+        body.topics.forEach((topic) => {
+          expect(topic).toMatchObject({
```

Comment:
Awesome test, love the use of to matchObject here to keep the tests flexible if the test data was to change!